university/models.py

```diff
@@ -16,7 +16,7 @@ class Professor(models.Model):
     website = models.CharField(max_length=256)
     picture = models.ImageField(blank=True, null=True, upload_to='profile_pictures')
 
-    previous_courses = models.ManyToManyField(Course, blank=True, null=True)
```

Comment:
doesn't setting blank=true require a default of null ? otherwise you can submit blank values in forms but it will error in the DB?